website/docs/API/basic-interface-overview.md

```diff
@@ -201,7 +201,7 @@ interface Config {
 
 interface PilotOptions {
   /** Cache mode (default: 'full') */
-  cacheMode?: CacheMode;  // 'full' | 'lightweight' | 'disabled'
+  cacheMode?: CacheMode;  // 'full' | 'disabled'
```

Comment:
see my comment (`isCacheEnabled`) - we have two modes now and it quite fit to boolean